editor/editor_node.cpp

```diff
@@ -7858,6 +7867,9 @@ EditorNode::EditorNode() {
 	project_menu->add_shortcut(ED_SHORTCUT_AND_COMMAND("editor/project_settings", TTRC("Project Settings..."), Key::NONE, TTRC("Project Settings")), PROJECT_OPEN_SETTINGS);
 	project_menu->connect(SceneStringName(id_pressed), callable_mp(this, &EditorNode::_menu_option));
 
+	ED_SHORTCUT_AND_COMMAND("editor/find_in_files", TTRC("Find in Files..."), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::F);
```

Comment:
```suggestion
	ED_SHORTCUT_AND_COMMAND("editor/find_in_files", TTRC("Find in Files"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::F);
```
To keep it the same as in script editor.

Reply:
@Frozenfire92 I tested it again and looks like suggested change is completely useless... Sorry about that. But even with this change everything still works fine.

Reply:
@arkology it worked for me 🤔 

![image](https://github.com/user-attachments/assets/f6d83cd2-be5d-45eb-aea9-226518f59a56)

![image](https://github.com/user-attachments/assets/247fe09c-d72d-418b-be55-c24eaf5e9442)

Reply:
Ah ok, my misunderstanding 😅

Reply:
I ended up going back to as it was to keep things consistent between locations